### PR TITLE
Tel 9803 debug crypto

### PR DESCRIPTION
**Pull request by piotrgregor**

### CODE REVIEW

File: src/switch_core_media.c [hunk 1]
```diff
@@ -1893,7 +1894,7 @@ SWITCH_DECLARE(int) switch_core_session_check_incoming_crypto(switch_core_sessio
 	if (engine->ssec[engine->crypto_type].remote_crypto_key && switch_rtp_ready(engine->rtp_session)) {
 		/* Compare all the key. The tag may remain the same even if key changed */
 		if (crypto && engine->crypto_type != CRYPTO_INVALID && !strcmp(crypto, engine->ssec[engine->crypto_type].remote_crypto_key) && sdp_type == SDP_TYPE_RESPONSE) {
-			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Existing key is still valid.\n");
+			switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Existing key is still valid. (sdp type %s)\n", SDP_TYPE_RESPONSE ? "SDP_TYPE_RESPONSE" : "SDP_TYPE_REQUEST");
```
Comment (dev-ryanc) [inline src/switch_core_media.c:1897]: this should be `sdp_type == SDP_TYPE_RESPONSE ? "SDP_TYPE_RESPONSE" : "SDP_TYPE_REQUEST"`

File: src/switch_core_media.c [hunk 2]
```diff
@@ -6022,6 +6023,7 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 
 					crypto = attr->a_value;
 					crypto_tag = atoi(crypto);
+					switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Checking incoming crypto\n");
```
Comment (dev-ryanc) [inline src/switch_core_media.c:6026]: I think we should add more info on this log such as the result on the switch_core_session_check_incoming_crypto

File: src/switch_core_media.c [hunk 3]
```diff
@@ -6612,6 +6614,7 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 					crypto = attr->a_value;
 					crypto_tag = atoi(crypto);
 
+					switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Checking incoming crypto\n");
```
Comment (dev-ryanc) [inline src/switch_core_media.c:6617]: I think we should add more info on this log such as the result on the switch_core_session_check_incoming_crypto

File: src/switch_core_media.c [hunk 4]
```diff
@@ -6766,6 +6769,7 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
...
```
Comment (dev-ryanc) [inline src/switch_core_media.c:6772]: I think we should add more info on this log such as the result on the switch_core_session_check_incoming_crypto

**Review by dev-ryanc — CHANGES_REQUESTED** [review]

File: src/switch_core_media.c (hunk 1 above)
Comment (piotrgregor) [reply src/switch_core_media.c]: Sure, thanks.

File: src/switch_core_media.c (hunk 3 above)
Comment (piotrgregor) [reply src/switch_core_media.c:6617]: If result is negative warning are printed as normally. Apart from that I added some logging inside switch_core_session_check_incoming_crypto. To be honest the purpose of this one is just to tell when the switch_core_session_check_incoming_crypto is executed.

File: src/switch_core_media.c (hunk 4 above)
Comment (piotrgregor) [reply src/switch_core_media.c:6772]: If result is negative warning are printed as normally. Apart from that I added some logging inside switch_core_session_check_incoming_crypto. To be honest the purpose of this one is just to tell when the switch_core_session_check_incoming_crypto is executed.

File: src/switch_core_media.c (hunk 2 above)
Comment (piotrgregor) [reply src/switch_core_media.c:6026]: If result is negative warning are printed as normally. Apart from that I added some logging inside switch_core_session_check_incoming_crypto. To be honest the purpose of this one is just to tell when the switch_core_session_check_incoming_crypto is executed.

**Review by dev-ryanc — APPROVED** [review]

**Pull request merged**